src/utils/entity-id.ts

```diff
@@ -0,0 +1,5 @@
+import {uniqueId} from 'src/lodash';
```

Comment:
nit
`uniqueId` is just a sequential number, maybe `uuid` is better?

Reply:
yeah, changed to uuid v4